recipes/mne-python_1.2/construct.yaml

```diff
@@ -74,6 +74,8 @@ specs:
   - mne-ari =0.1.1
   - mne-kit-gui =1.0.1
   - mne-icalabel =0.4  # [not win]
+  - mnelab =0.8.5   # [not arm64]  # https://github.com/conda-forge/pyside2-feedstock/issues/159
+  - pyside6 =6.4.1  # [not arm64]  # For MNELAB. Without this pinning, 6.4.0 might get installed (why?!)
```

Comment:
```suggestion
  # https://github.com/conda-forge/pyside2-feedstock/issues/159
  - mnelab =0.8.5   # [not arm64]
  # For MNELAB. Without this pinning, 6.4.0 might get installed (why?!)
  - pyside6 =6.4.1  # [not arm64]
```

Reply:
That was the issue? D'oh! 😅 thanks for figuring it out!!!